fix the solution 8025 error (#18152)
pingcap/docs: tidb-lightning/troubleshoot-tidb-lightning.md

**Solution**:

> - Use the system variable [`tidb_txn_entry_size_limit`](/system-variables.md#tidb_txn_entry_size_limit-new-in-v760) to dynamically increase the limit.

Reviewer: ```suggestion
- Use the [`tidb_txn_entry_size_limit`](/system-variables.md#tidb_txn_entry_size_limit-new-in-v760) system variable to dynamically increase the limit.
```

> - Use the system variable `tidb_txn_entry_size_limit` to dynamically increase the limit
> - Note that TiKV has a similar limit. If the data size of a single write request exceeds raft-entry-max-size, which is 8 MB by default, TiKV refuses to process this request. When a table has a row of large size, you need to modify both configurations at the same time.

Reviewer 2: ```suggestion
- Use the system variable [`tidb_txn_entry_size_limit`](/stable/system-variables.md#tidb_txn_entry_size_limit-new-in-v760) to dynamically increase the limit.
- Note that TiKV has a similar limit. If the data size of a single write request exceeds [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size) (`8MiB` by default), TiKV refuses to process this request. When a table has a row of a large size, you need to modify both configurations at the same time.
```

> - Note that TiKV has a similar limit. If the data size of a single write request exceeds [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size) (`8MiB` by default), TiKV refuses to process this request. When a table has a row of a large size, you need to modify both configurations at the same time.

Reviewer: ```suggestion
- Note that TiKV has a similar limit. If the data size of a single write request exceeds [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size) (`8MiB` by default), TiKV refuses to process this request. When a table has a row of a large size, you need to modify both configurations.
```
